### PR TITLE
Move display & MIME stuff out of helpDB, xrefs

### PR DESCRIPTION
**Pull request by kshyatt**

### CODE REVIEW

**Review by KristofferC — APPROVED** [review]

**Review by fredrikekre — APPROVED** [review]

File: base/docs/helpdb/Base.jl [hunk 1]
```diff
@@ -423,24 +407,19 @@ Show every part of the representation of a value.
...
 """
-    display(x)
-    display(d::Display, x)
-    display(mime, x)
-    display(d::Display, mime, x)
+    isinteractive() -> Bool
+
+Determine whether Julia is running an interactive session.
+"""
+isinteractive
```
Comment (tkelman) [inline base/docs/helpdb/Base.jl]: did this just get moved?

Comment (kshyatt) [inline base/docs/helpdb/Base.jl]: I'm really confused about why github is showing a merge conflict. If I look at the files changed tab, it matches what I have on my desktop, which has `isinteractive` still in helpdb...

Comment (kshyatt) [inline base/docs/helpdb/Base.jl]: Oh I see! Yes it appears this got put back! Ugh, sorry!

**Comment by kshyatt** [comment]
OK!! IDK what happened there but it seems the conflict is resolved. Make sure to squash this, whoever merges it.

**Review by Sacha0 — APPROVED** [review]

**Pull request merged**